examples/dynamics/oscillator.ipynb

```diff
@@ -154,4 +156,4 @@
    "metadata": {}
   }
  ]
-}
```

Comment:
In some of these files, the endline at the end of the file has been removed, and in others it's been added. We should probably decide which way we want it, and then change the cleanup devscript to do it that way. I think when I wrote the devscript, I put in the linebreak at the end because I don't like the little :no_entry_sign: that GitHub puts if you don't have it (command line git also gets mad), but I think IPython doesn't put one when you edit a file (and will even actively remove it), so perhaps we should go with no linebreak.